test/system/090-events.bats

```diff
@@ -291,3 +291,22 @@ EOF
     _events_container_create_inspect_data journald
     _events_container_create_inspect_data file
 }
+
+@test "event with too big labels" {
+    # test that events entries are not truncated when reading from syslog
+
+    skip_if_remote "remote does not support --events-backend"
+
+    cname=test-$(random_string 30 | tr A-Z a-z)
+
+    local label_file="$BATS_TEST_TMPDIR/labels.txt"
+    echo -n "too-long-label=" >> "$label_file"
+    random_string 150000 >> "$label_file"
```

Comment:
Maybe it's my old age, but I get squicked out by things like this, because my gut screams that it will block waiting for entropy. Maybe that's not a problem these days? Anyhow, I see little reason to find out. This doesn't really have to be a random string. But I won't block over it.

Reply:
It would block on `/dev/random` but not `/dev/urandom` I think.

Reply:
fixed

Reply:
I just just Good Enough™ approach with loop. Is there more more efficient way?